src/DebianTarballBuilder.cs

```diff
@@ -83,54 +83,96 @@ private async Task<bool> ProcessDirectoryAsync(
                 return false;
             }
         }
-        
-        foreach (var (fileName, (fileInfo, _)) in destinationFiles)
+
+        foreach (var (fileName, (fileInfo, _, isFlamencoFile)) in destinationFiles)
         {
             cancellationToken.ThrowIfCancellationRequested();
-                
-            var destinationPath = Path.Combine(destinationDirectory.FullName, fileName);
 
-            try
+            if (!isFlamencoFile)
             {
-                fileInfo.CopyTo(destinationPath);
+                var destinationPath = Path.Combine(destinationDirectory.FullName, fileName);
+
+                try
+                {
+                    fileInfo.CopyTo(destinationPath);
+                }
+                catch (Exception exception)
+                {
+                    Log.Error($"Failed to create destination file '{destinationPath}'");
+                    Log.Debug($"Exception Message: {exception.Message}");
+                    return false;
+                }
             }
-            catch (Exception exception)
+            else
             {
-                Log.Error($"Failed to create destination file '{destinationPath}'");
-                Log.Debug($"Exception Message: {exception.Message}");
-                return false;
+                switch (fileName)
+                {
+                    case "flamenco.external":
```

Comment:
issue: This is a constant value that is defined here and in `NamingConventions.ValidFlamencoFileNames`. This is an anti-pattern, because someone could edit just the value in one place and forget to edit the other.